app/main/forms.py

```diff
@@ -1096,10 +1096,18 @@ class AdminServiceSMSAllowanceForm(StripWhitespaceForm):
 
 
 class AdminServiceMessageLimitForm(StripWhitespaceForm):
-    message_limit = GovukIntegerField(
-        "Number of messages the service is allowed to send each day",
-        validators=[DataRequired(message="Cannot be empty")],
-    )
+    message_limit = GovukIntegerField("", validators=[DataRequired(message="Cannot be empty")])
+
+    def __init__(self, notification_type=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        limit = notification_type or "message"
```

Comment:
`from app.formatters import message_count_noun` then:
```suggestion
        limit = message_count_noun(1, notification_type)
```

Reply:
🤯